site/dat/docs/Apiritif.md

```diff
@@ -475,6 +526,15 @@ For storing variables use the following actions:
 
 See documentation for `X` [here](#Locators).
 
+For storing the result of evaluation of a JavaScript expression use:
+
+- `storeEval(js_expr): var_eval`
+
+Note that the result of the evaluation will be converted to String data type.
```

Comment:
it isn't true according to my check. just try to store number and then check its type

Reply:
You're right... I will update the doc.